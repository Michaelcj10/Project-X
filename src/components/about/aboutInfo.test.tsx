import * as React from "react";
import { render } from "@testing-library/react";
import AboutInfo from "./aboutInfo";

describe("<AboutInfo /> spec", () => {
  it("renders the component", () => {
    const container = render(<AboutInfo />);
    expect(container).toMatchSnapshot();
  });
});
