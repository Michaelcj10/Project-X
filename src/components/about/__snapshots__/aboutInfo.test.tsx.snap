// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutInfo /> spec renders the component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
        <div
            class="sc-AykKD dEvrDz"
        >
            <div
                class="about-block"
            >
                <div
                    class="row"
                >
                    <div
                        class="twelve columns centered"
                    >
                        <div>
                            <h2>
                                We give your site
                            </h2>
                        </div>
                    </div>
                </div>
                <div
                    class="row"
                >
                    <div>
                        <div
                            class="one-third column centered"
                        >
                            <svg
                                viewBox="0 0 24 24"
                                width="50"
                                aria-hidden="true"
                                focusable="false"
                                fill="currentColor"
                                xmlns="http://www.w3.org/2000/svg"
                                class="sc-AykKC eYrmsm"
                            >
                                <path
                                    fill="none"
                                    d="M0 0h24v24H0V0z"
                                />
                                <path
                                    d="M15 1H9v2h6V1zm4.03 6.39l1.42-1.42c-.43-.51-.9-.99-1.41-1.41l-1.42 1.42A8.962 8.962 0 0012 4c-4.97 0-9 4.03-9 9s4.02 9 9 9a8.994 8.994 0 007.03-14.61zM12 20c-3.87 0-7-3.13-7-7s3.13-7 7-7 7 3.13 7 7-3.13 7-7 7zm-.32-5H6.35a5.992 5.992 0 003.41 3.56l-.11-.06 2.03-3.5zm5.97-4a6.012 6.012 0 00-3.34-3.54L12.26 11h5.39zm-7.04 7.83c.45.11.91.17 1.39.17 1.34 0 2.57-.45 3.57-1.19l-2.11-3.9-2.85 4.92zM7.55 8.99A5.965 5.965 0 006 13c0 .34.04.67.09 1h4.72L7.55 8.99zm8.79 8.14A5.94 5.94 0 0018 13c0-.34-.04-.67-.09-1h-4.34l2.77 5.13zm-3.01-9.98C12.9 7.06 12.46 7 12 7c-1.4 0-2.69.49-3.71 1.29l2.32 3.56 2.72-4.7z"
                                />
                            </svg>
                            <div
                                class="description"
                            >
                                Speed
                            </div>
                        </div>
                    </div>
                    <div>
                        <div
                            class="one-third column centered"
                        >
                            <svg
                                viewBox="0 0 24 24"
                                width="50"
                                aria-hidden="true"
                                focusable="false"
                                fill="currentColor"
                                xmlns="http://www.w3.org/2000/svg"
                                class="sc-AykKC eYrmsm"
                            >
                                <path
                                    d="M17 2H7c-1.103 0-2 .897-2 2v16c0 1.103.897 2 2 2h10c1.103 0 2-.897 2-2V4c0-1.103-.897-2-2-2zM7 16.999V5h10l.002 11.999H7z"
                                />
                            </svg>
                            <div
                                class="description"
                            >
                                Mobile Responsivness
                            </div>
                        </div>
                    </div>
                    <div>
                        <div
                            class="one-third column centered"
                        >
                            <svg
                                viewBox="0 0 24 24"
                                width="50"
                                aria-hidden="true"
                                focusable="false"
                                fill="currentColor"
                                xmlns="http://www.w3.org/2000/svg"
                                class="sc-AykKC eYrmsm"
                            >
                                <path
                                    d="M17 21H7c-1.654 0-3-1.346-3-3V6c0-1.654 1.346-3 3-3h10c1.654 0 3 1.346 3 3v12c0 1.654-1.346 3-3 3zM7 5c-.551 0-1 .449-1 1v12c0 .551.449 1 1 1h10c.551 0 1-.449 1-1V6c0-.551-.449-1-1-1H7zm9 6H8a.5.5 0 010-1h8a.5.5 0 010 1zm0-3H8a.5.5 0 010-1h8a.5.5 0 010 1zm0 6H8a.5.5 0 010-1h8a.5.5 0 010 1zm0 3H8a.5.5 0 010-1h8a.5.5 0 010 1z"
                                />
                            </svg>
                            <div
                                class="description"
                            >
                                SEO Compliance
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</body>,
  "container": <div>
    <div
        class="sc-AykKD dEvrDz"
    >
        <div
            class="about-block"
        >
            <div
                class="row"
            >
                <div
                    class="twelve columns centered"
                >
                    <div>
                        <h2>
                            We give your site
                        </h2>
                    </div>
                </div>
            </div>
            <div
                class="row"
            >
                <div>
                    <div
                        class="one-third column centered"
                    >
                        <svg
                            viewBox="0 0 24 24"
                            width="50"
                            aria-hidden="true"
                            focusable="false"
                            fill="currentColor"
                            xmlns="http://www.w3.org/2000/svg"
                            class="sc-AykKC eYrmsm"
                        >
                            <path
                                fill="none"
                                d="M0 0h24v24H0V0z"
                            />
                            <path
                                d="M15 1H9v2h6V1zm4.03 6.39l1.42-1.42c-.43-.51-.9-.99-1.41-1.41l-1.42 1.42A8.962 8.962 0 0012 4c-4.97 0-9 4.03-9 9s4.02 9 9 9a8.994 8.994 0 007.03-14.61zM12 20c-3.87 0-7-3.13-7-7s3.13-7 7-7 7 3.13 7 7-3.13 7-7 7zm-.32-5H6.35a5.992 5.992 0 003.41 3.56l-.11-.06 2.03-3.5zm5.97-4a6.012 6.012 0 00-3.34-3.54L12.26 11h5.39zm-7.04 7.83c.45.11.91.17 1.39.17 1.34 0 2.57-.45 3.57-1.19l-2.11-3.9-2.85 4.92zM7.55 8.99A5.965 5.965 0 006 13c0 .34.04.67.09 1h4.72L7.55 8.99zm8.79 8.14A5.94 5.94 0 0018 13c0-.34-.04-.67-.09-1h-4.34l2.77 5.13zm-3.01-9.98C12.9 7.06 12.46 7 12 7c-1.4 0-2.69.49-3.71 1.29l2.32 3.56 2.72-4.7z"
                            />
                        </svg>
                        <div
                            class="description"
                        >
                            Speed
                        </div>
                    </div>
                </div>
                <div>
                    <div
                        class="one-third column centered"
                    >
                        <svg
                            viewBox="0 0 24 24"
                            width="50"
                            aria-hidden="true"
                            focusable="false"
                            fill="currentColor"
                            xmlns="http://www.w3.org/2000/svg"
                            class="sc-AykKC eYrmsm"
                        >
                            <path
                                d="M17 2H7c-1.103 0-2 .897-2 2v16c0 1.103.897 2 2 2h10c1.103 0 2-.897 2-2V4c0-1.103-.897-2-2-2zM7 16.999V5h10l.002 11.999H7z"
                            />
                        </svg>
                        <div
                            class="description"
                        >
                            Mobile Responsivness
                        </div>
                    </div>
                </div>
                <div>
                    <div
                        class="one-third column centered"
                    >
                        <svg
                            viewBox="0 0 24 24"
                            width="50"
                            aria-hidden="true"
                            focusable="false"
                            fill="currentColor"
                            xmlns="http://www.w3.org/2000/svg"
                            class="sc-AykKC eYrmsm"
                        >
                            <path
                                d="M17 21H7c-1.654 0-3-1.346-3-3V6c0-1.654 1.346-3 3-3h10c1.654 0 3 1.346 3 3v12c0 1.654-1.346 3-3 3zM7 5c-.551 0-1 .449-1 1v12c0 .551.449 1 1 1h10c.551 0 1-.449 1-1V6c0-.551-.449-1-1-1H7zm9 6H8a.5.5 0 010-1h8a.5.5 0 010 1zm0-3H8a.5.5 0 010-1h8a.5.5 0 010 1zm0 6H8a.5.5 0 010-1h8a.5.5 0 010 1zm0 3H8a.5.5 0 010-1h8a.5.5 0 010 1z"
                            />
                        </svg>
                        <div
                            class="description"
                        >
                            SEO Compliance
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
